spec/javascripts/packages/document-capture/components/submission-complete-spec.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import sinon from 'sinon';
+import { waitFor } from '@testing-library/dom';
+import useAsync from '@18f/identity-document-capture/hooks/use-async';
+import SubmissionComplete, {
+  RetrySubmissionError,
+} from '@18f/identity-document-capture/components/submission-complete';
+import SuspenseErrorBoundary from '@18f/identity-document-capture/components/suspense-error-boundary';
+import render from '../../../support/render';
+import { useCleanDOM } from '../../../support/dom';
+import { useSandbox } from '../../../support/sinon';
+
+describe('document-capture/components/submission-complete-step', () => {
+  const sandbox = useSandbox();
+  useCleanDOM();
+
+  let response = { success: true };
+
+  function TestComponent() {
+    const resource = useAsync(() => Promise.resolve(response));
+    return <SubmissionComplete resource={resource} />;
+  }
+
+  it('renders fallback while loading', () => {
+    const { getByText } = render(
+      <SuspenseErrorBoundary fallback="Loading...">
+        <TestComponent />
+      </SuspenseErrorBoundary>,
+    );
+
+    expect(getByText('Loading...')).to.be.ok();
+  });
+
+  it('submits form once loading is complete', async () => {
+    const onSubmit = sinon.mock().callsFake((event) => event.preventDefault());
+
+    const form = document.createElement('form');
+    form.className = 'js-document-capture-form';
+    form.addEventListener('submit', onSubmit);
+    document.body.appendChild(form);
```

Comment:
We seem to have this form pattern a lot, should we simplify it with a helper or something?